.gitignore

```diff
@@ -39,3 +39,4 @@ models.cfg
 pythia*.tgz
 src/chromo/iamdata
 fig/*
+/src/f2py/
```

Comment:
I don't think we need to ignore this, the directory can be removed completely, no?

Reply:
I resolved this in my patch.